docs/product-portal/applications/consuming-megazord-libraries.md

```diff
@@ -91,11 +91,12 @@ to fit the needs of existing Firefox applications:
 | `lockbox` | `fxaclient`, `logins` | `org.mozilla.appservices:lockbox-megazord` |
 | `fenix` | `fxaclient`, `logins`, `places` | `org.mozilla.appservices:fenix-megazord` |
 
-Then, simply use gradle's builtin support for [module replacement](https://docs.gradle.org/current/userguide/customizing_dependency_resolution_behavior.html#sec:module_replacement)
+Then, use gradle's builtin support for [module replacement](https://docs.gradle.org/current/userguide/customizing_dependency_resolution_behavior.html#sec:module_replacement)
 to replace the default "full megazord" with your selected custom build:
 
 ```groovy
 dependencies {
+  implementation 'org.mozilla.appservices:fenix-megazord:X.Y.Z'
```

Comment:
This was previously buggy, you have to actually declare the dependency on the custom megazord in order to get the `replacedBy` rule to work.